src/commands/createTransactionCreateMultisignatureAccount.js

```diff
@@ -75,7 +75,7 @@ export const actionCreator = vorpal => async ({ lifetime, minimum, keysgroup, op
 			repeatPrompt: true,
 		},
 	})
-		.then(createMultisignatureAccount(
+		.then(handleInputs(
```

Comment:
`handleInputs` is a little too generic for me. It seems to me we have handled the inputs and now proceed with the actual action. So how about e.g. `executeAction`?

Reply:
The way I was thinking of it, the steps up until this point have been about *gathering* the inputs, and now that they're gathered they can be *handled*. I think `executeAction` is confusing because we're already in the action; the action covers gathering and handling inputs.

I'm open to a more illuminating name though. Maybe something along the lines of `processInputs`?

Reply:
Yes, `processInputs` seems fine to me 👍